src/app/pages/datasets/components/dataset-details-card/dataset-details-card.component.html

```diff
@@ -65,6 +65,7 @@ <h3 mat-card-title *ngIf="isZvol">{{ 'Zvol Details' | translate }}</h3>
           matTooltipPosition="above"
           class="value dataset-path"
           [matTooltip]="dataset.name"
+          [matTooltipClass]="['tooltip-name']"
```

Comment:
I didn't mean to call it .tooltip-name

it's a placeholder, please call it **dataset-path-tooltip**